ironfish/src/network/messages/getBlockHeaders.ts

```diff
@@ -0,0 +1,93 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import bufio from 'bufio'
+import { BlockHeader } from '../../primitives'
+import { NetworkMessageType } from '../types'
+import { getBlockHeaderSize, readBlockHeader, writeBlockHeader } from '../utils/serializers'
+import { Direction, RpcNetworkMessage } from './rpcNetworkMessage'
+
+export class GetBlockHeadersRequest extends RpcNetworkMessage {
+  readonly start: number
+  readonly limit: number
+  readonly skip: number
+  readonly reverse: boolean
+
+  constructor(start: number, limit: number, skip: number, reverse: boolean, rpcId?: number) {
+    super(NetworkMessageType.GetBlockHeadersRequest, Direction.Request, rpcId)
+    this.start = start
```

Comment:
This is my fault for not noticing earlier, but the GetBlockHeaders method in eth takes either a sequence or a blockHash for the start value: https://github.com/ethereum/devp2p/blob/master/caps/eth.md#getblockheaders-0x03

I'm not sure that we use both now, but could we add that? The main benefit I think would be if the chain forks and rolls back while you're fetching block headers, then the sequences may continue to match up but not connect with the previous fetched headers.